packages/server/src/service/pdfForm/newHampshire.ts

```diff
@@ -27,7 +27,7 @@ export const fillNewHampshire = (
     text(stateInfo.email, 1, 250, 300)
     text(new Date().toISOString().split('T')[0], 1, 480, 340)
 
-    const signatureBuffer = await toSignatureBuffer(stateInfo.signature, 200, 50)
-    await placeImage(new Uint8Array(signatureBuffer.buffer), 1, 250, 360)
+    const signatureBuffer = await toSignatureBuffer(stateInfo.signature, 150, 30)
```

Comment:
Why do we have to shrink the image size?  200 x 50 px is already pretty small.  Is that too big for the form?

Reply:
Hmm yeah, I mean, greater than 120 on North Carolina was overlapping with the date.  You can see New Hampshire's max height here:

<img width="805" alt="Screen Shot 2020-06-26 at 3 42 15 PM" src="https://user-images.githubusercontent.com/8974801/85894944-ac925b80-b7c3-11ea-812d-ef2c6575cb16.png">

I dunno how much outside of the box you'd want it to be, but note that the trim adjusts the image nicely, so if it's long horizontally and short vertically, it will in fact take up the full 150x30 space.

Here's an example of this with NC and NH:
<img width="669" alt="Screen Shot 2020-06-26 at 3 44 03 PM" src="https://user-images.githubusercontent.com/8974801/85895077-f1b68d80-b7c3-11ea-89a0-a2b0f954b743.png">
<img width="761" alt="Screen Shot 2020-06-26 at 3 44 21 PM" src="https://user-images.githubusercontent.com/8974801/85895078-f1b68d80-b7c3-11ea-8939-6755ea9295a4.png">

Reply:
The latter 2 images in the comments above are me pushing the width, the first image is me pushing the height on NH.  I did the same for height on NC.

Reply:
Thanks!